media-library/overview/search-update-and-delete.md

```diff
@@ -32,6 +32,39 @@ You can further narrow search results by using quick filters `Folders`, `Files`,
 
 The toggle at the top, "Include files with matching tags," allows you to control whether the search results should include files with matching tags.
 
+### Visual Search
+
+{% hint style="info" %}
+**Enterprise plan only**
+
+This feature is only available in custom enterprise pricing plans.
+{% endhint %}
+
+Visual search allows you to search for similar images based on visual content. Instead of relying on text-based keywords or metadata,  you can search based on visual content like colors, shapes, textures and any other information that can be derived from the image itself.
+
+By searching visually rather than by name or metadata, you can:
+
+* Easily locate the images without needing to remember the tags or name of the file.
+* Identify duplicates at ease
+* Discover images in your media library through visual characteristics
+* Group visually similar images together for better organization
+* Enhance discoverability of your assets
+
+You can perform visual search from the dashboard either by image or by text:
```

Comment:
Also add a section for find similar images.
Talk about limitations on file types: image and mimes,  not supported for videos and non-image files.